docs/source/aws_provider.rst

```diff
@@ -0,0 +1,134 @@
+Adding an AWS Account
+#####################
+
+This following describes the steps necessary to configure an AWS account for cost and usage reporting to an S3 bucket, creation of an IAM Policy and Role to be utilized by Koku to process the cost usage data, and AWS organization setup in order to control cost visibility.
```

Comment:
This is really multiple sentences worth of content. Let's break this up and make it more clear.

`This document describes how to configure your AWS account to allow Koku access. Configuring your account involves configuring three AWS services. [one sentence describing what we'll do with each service]`

Reply:
Updated.